### PR TITLE
Add method to create accounts

### LINKED ISSUE #26: Account creation
Product requires the client to provide a method that creates an account with enough money to do everything. The method should return the address of the account.

For now we’ll use the account management facilities of the Parity Ethereum node. The account should be created with an empty password.

### PR DESCRIPTION
**Pull request by geigerzaehler**
Add a method that creates a new account. Fixes #26

### CODE REVIEW

File: client/src/lib.rs [hunk 1]
```diff
@@ -220,3 +224,6 @@ impl<'a> Future for SubmitResult<'a> {
         self.future.poll()
     }
 }
+
+/// [Future] for API call results with error [web3::error::Error]
```
Comment (rockbmb) [inline client/src/lib.rs]: Hm, why is this needed? Can't quite understand.

Comment (geigerzaehler) [reply client/src/lib.rs]: You mean the type alias? It is just provided for convenience. The client is not polymorphic in the transport so to simplify the types we fix the transport to `Http`.
